homeassistant/components/growatt_server/sensor.py

```diff
@@ -56,77 +57,94 @@
 }
 
 INVERTER_SENSOR_TYPES = {
-    "inverter_energy_today": ("Energy today", ENERGY_KILO_WATT_HOUR, "e_today", {},),
+    "inverter_energy_today": (
+        "Energy today",
+        ENERGY_KILO_WATT_HOUR,
+        "powerToday",
+        {"round": 1},
+    ),
     "inverter_energy_total": (
         "Lifetime energy output",
         ENERGY_KILO_WATT_HOUR,
-        "e_total",
-        {},
+        "powerTotal",
+        {"round": 1},
     ),
-    "inverter_voltage_input_1": ("Input 1 voltage", VOLT, "vpv1", {}),
+    "inverter_voltage_input_1": ("Input 1 voltage", VOLT, "vpv1", {"round": 2}),
     "inverter_amperage_input_1": (
         "Input 1 Amperage",
         ELECTRICAL_CURRENT_AMPERE,
         "ipv1",
-        {},
+        {"round": 1},
     ),
     "inverter_wattage_input_1": (
         "Input 1 Wattage",
         POWER_WATT,
         "ppv1",
-        {"device_class": "power"},
+        {"device_class": "power", "round": 1},
```

Comment:
Please use the device class constants from the sensor integration. Import those and use instead of defining a new string, eg `DEVICE_CLASS_POWER`.